docs/resources/mcs_db_cluster.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: "mcs"
+page_title: "mcs: db_cluster"
+sidebar_current: "docs-db-cluster"
+subcategory: ""
+description: |-
+  Manages a db cluster.
+---
+
+# mcs\_db\_cluster (Resource)
+
+Provides a db cluster resource. This can be used to create, modify and delete db cluster for galera_mysql and postgresql datastores.
+
+## Example Usage
+
+```terraform
+
+resource "mcs_db_cluster" "mydb-cluster" {
+  name      = "mydb-cluster"
+  flavor_id = example_flavor_id
+  volume_size      = 10
+  volume_type = example_volume_type
+
+  cluster_size = 3
+
+  network {
+    uuid = example_network_id
+  }
+
+  datastore {
+    version = "12"
+    type    = "postgresql"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the cluster. Changing this creates a new cluster
+
+* `volume_size` - (Required) Size of the cluster instance volume.
+
+* `volume_type` - (Required) The type of the cluster instance volume. Changing this creates a new cluster.
+
+* `flavor_id` - (Required) The ID of flavor for the cluster.
+
+* `datastore` - (Required) Object that represents datastore of the cluster. Changing this creates a new cluster. It has following attributes:
```

Comment:
move this up to name to make structure of both cluster and cluster with shards more similar

Reply:
Fixed

Reply:
Now the order looks better but still not consistent.
1 Orders of keys in instance and cluster differs from each other
2 Keys related to VM need to be grouped (flavor, AZ, keypair, public access)
3 So do volume related keys (size, type, autoexpand, wal)

Reply:
Fixed